db/products-controllers.ts

```diff
@@ -0,0 +1,94 @@
+import Product from './products';
+import * as productValidation from '../config/products-validation';
+import {logger} from '../config/logger';
+
+class ProductsControllers {
+    async getProducts(ctx) {
+        ctx.ok(await Product.find().select('-__v'));
```

Comment:
dont remove __v like that. if you dont need it at all change product mongoose schema so it wont create it